### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/locales/pt/app.json [hunk 1]
```diff
@@ -15,20 +15,30 @@
     },
     "initializing": "Obtendo informações da blockchain...",
     "maybeProxyError": "Tor proxy ativado. Certifique-se que seu Tor esteja rodando corretamente ou desative as opções de proxy.",
-    "openFile": "Abrir arquivo",
+    "openFile": "",
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/pt/app.json:18]: idk, @jstrnbrg ?

File: frontends/web/src/locales/pt/app.json [hunk 2]
```diff
@@ -39,7 +49,8 @@
     "fiatBalance": "Saldo em moeda corrente",
     "name": "Nome da conta",
     "noAccount": "Não há contas para mostrar.",
-    "synchronizing": "Sincronizando contas com suas blockchains:",
+    "subtotalWithCoinName": "Total ({{coinName}})",
+    "synchronizing": "",
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/pt/app.json:53]: It was removed in cea8aeb6257fe4402320a2e6533ae921fc0cf1db. I am not sure why locize doesn't delete the keys in the other languages. Need to check, but not a blocker here.

File: frontends/web/src/locales/pt/app.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/pt/app.json:18]: Same as https://github.com/digitalbitbox/bitbox-wallet-app/pull/1717#discussion_r901583316, the key was deleted in `en`.

**Comment by benma** [comment]
Rebased and pulled again

File: frontends/web/src/locales/de/app.json [hunk 3]
```diff
@@ -1217,6 +996,75 @@
...
+      "message": "Bestätige auf deiner BitBox, dass du die optionale Passphrase <strong>aktivieren</strong> möchtest.",
+      "title": "Bestätige am Gerät"
+    },
+    "successDisabled": {
+      "message": "Optionale Passphrase <strong>erfolgreich aktiviert</strong>!\nVon nun an wirst du aufgefordert, eine Passphrase anzugeben.",
+      "messageEnd": "Bitte stecke die BitBox02 jetzt wieder ein.",
+      "title": "Passphrase aktiviert"
+    },
+    "successEnabled": {
+      "message": "Optionale Passphrase <strong>erfolgreich deaktiviert</strong>!\nVon nun an wirst du aufgefordert, eine Passphrase anzugeben.",
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:1037]: This is wrong, when it is successfully deactivated it should say "Von nun an wirst du nicht mehr aufgefordert eine Passphrase anzugeben." or similar.

File: frontends/web/src/locales/de/app.json [hunk 4]
```diff
@@ -1229,16 +1077,19 @@
     "description": "Ihre BitBox hat die folgende {{bits}}-Bit-Zufallszahl generiert:"
   },
   "receive": {
-    "addressType": [
-      "Zu kompatibler Adresse ändern (Segwit)",
-      "Zurück zum neuen Adressformat (empfohlen)"
-    ],
+    "changeScriptType": "Addresstyp ändern",
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:1080]: Adresstyp

**Review by thisconnect — CHANGES_REQUESTED** [review]
Find some inconsistencies and opened a few PRs

Also in the is a wrong sentence in the passphrase-disable-success-message

File: frontends/web/src/locales/de/app.json (hunk 3 above)
Comment (thisconnect) [reply frontends/web/src/locales/de/app.json:1037]: cc @jstrnbrg

File: frontends/web/src/locales/de/app.json [hunk 5]
```diff
@@ -15,20 +15,29 @@
...
     "label": "Konto Informationen",
+    "scriptType": "Scripttyp",
     "title": "Konto Informationen",
     "verify": "Auf Gerät bestätigen",
     "xpubTypeChangeBtn": {
       "p2pkh": "Legacy-P2PKH erweiterter Public Key anzeigen",
+      "p2tr": "Taproot ansehen",
       "p2wpkh": "Native Segwit anzeigen",
-      "p2wpkh-p2sh": "Älteren Segwit erweiterte Public Key anzeigen"
+      "p2wpkh-p2sh": "Älteren Segwit erweiterten öffentlichen Schlüssel anzeigen"
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/de/app.json:40]: @thisconnect feel free to decide and fix on locize, I don't mind either way :)

**Comment by benma** [comment]
Pulled and rebased again

File: frontends/web/src/locales/it/app.json [hunk 6]
```diff
@@ -680,7 +680,7 @@
         "title": "Perché così tanti indirizzi?"
       },
       "whyVerify": {
-        "text": "Perché così tanti indirizzi?",
+        "text": "",
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/it/app.json:752]: Translated with help from SpaghettiMonster, @benma could you pull again?

[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/it/app.json:752]: this one would be nice other than that LGTM

File: frontends/web/src/locales/de/app.json [hunk 7]
```diff
@@ -39,44 +48,29 @@
     "fiatBalance": "Fiat Kontostand",
     "name": "Konto Name",
     "noAccount": "Keine Konten verfügbar",
+    "subtotalWithCoinName": "({{coinName}}) - Summe",
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/de/app.json]: I agree with @thisconnect and I fixed it on locize.

File: frontends/web/src/locales/de/app.json [hunk 8]
```diff
@@ -690,7 +491,8 @@
     "enable_false": "Deaktivieren",
     "enable_true": "Aktivieren",
     "enabled_false": "Deaktiviert",
-    "enabled_true": "Aktiviert"
+    "enabled_true": "Aktiviert",
+    "success": "Erfolg"
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/de/app.json:495]: The whole `generic` section seems unused. git blame shows it was used for the BitBoxBase: 

image

I added a commit to remove it.

**Comment by benma** [comment]
Rebased again :smile:

File: frontends/web/src/locales/de/app.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/de/app.json:40]: soon™

File: frontends/web/src/locales/de/app.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/de/app.json:495]: I see `generic.confirmToContinue` and `generic.enabled` is used, all others can be removed

[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/de/app.json:495]: Restored generic.enabled!

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**